### ISSUE TITLE
Report order cycle distributor totals by supplier gives wrong information in relation to delivery method

### ISSUE THREAD

**Issue by nicolasVr** [openfoodfoundation/openfoodnetwork#12877]
## Description
This is the only report that displays delivery method for order cycles. But the result is wrong as explained below.

## Expected Behavior
The report should give the total quantity for each delivery method : 
Product | Qtty | Delivery Method
A | 3 | Place X
A | 4 | Place Y

## Actual Behaviour
Instead of that there is only 1 line per product and I don't know how the delivery method displayed is chosen. We'll get for our example : 
Product | Qtty | Delivery Method
A | 7 | Place X

## Steps to Reproduce
Ask a report for an order cycle where a product has been ordered with 2 different delivery methods.

## Workaround
No report with this information !

## Severity
s3 ?

**Comment by chahmedejaz** [comment]
@nicolasVr - FYI, I just did some investigation on this one, and seems the report is intended to work as per the current behavior 😅. The issue's expected behavior makes it a feature request rather than a bug as almost half of this report needs to be rewritten. 

No worries, working on it 😄

**Comment by RachL** [comment]
@chahmedejaz Ahmed hold on I don't think we have budget for this one. Can you have a look at estimating the issues I've linked here instead ? https://openfoodnetwork.slack.com/archives/C01T75H6G0Z/p1728638876580479

**Comment by chahmedejaz** [comment]
Sure @RachL, Thanks for the headsup. I'd fixed the issue already, only its test cases remain 😅 . I've pushed the branch here for anyone to pick up for fixing the test cases: https://github.com/chahmedejaz/openfoodnetwork/tree/task/12877-fix-information-in-order-cycle
Here are the changes:
https://github.com/chahmedejaz/openfoodnetwork/commit/0a365c063ddc1eeb72d4742fcf4911a53fbf5f0c

For now, I'm unassigning for anyone to take it up. Thanks

**Comment by chahmedejaz** [comment]
Sure @RachL let me do that 👍

**Comment by BethanOFN** [comment]
We've got a user in the UK who has flagged this - they run 4 order cycles per week for different customer groups but collect/receive from producers once per week and also manage finances weekly. They therefore have to run the OC Distributor Totals by Supplier report 4 times each week, once for each cycle during the week, and then manually combine the reports so that they can work out their weekly totals split by OC. Shipping Method, if included as a report column, should split the results as Nicolas says, e.g. if Mint is ordered in two different order cycles with two different associated shipping methods, and two order cycles are selected in the report; the report should show _two rows for Mint, split by shipping method_.

**Comment by chahmedejaz** [comment]
Hi @nicolasVr @BethanOFN - I'm working on this issue and just wanted to confirm whether the summary row in the screenshot below is the expected behaviour.

<img width="1763" height="461" alt="Image" src="https://github.com/user-attachments/assets/f4fa3e60-080b-4749-8f62-73d41a45b379" />

The totals in the summary row are currently calculated per hub.

Moreover, I also noticed the shipping method issue exists in the **Order Cycle Supplier Totals by Distributor** report. If this is unintended, should I fix it there as well? Thanks!

**Comment by nicolasVr** [comment]
Yes, the orders are split by shipping method and not randomly grouped in the same shipping method. It is right for me.

> Moreover, I also noticed the shipping method issue exists in the Order Cycle Supplier Totals by Distributor report. If this is unintended, should I fix it there as well? Thanks!

I had not noticed that this report also displays Shipping Method. I think it should work the same way.
